binderhub/builder.py

```diff
@@ -308,6 +310,7 @@ def escape(s):
             docker_host=self.settings['build_docker_host'],
             node_selector=self.settings['build_node_selector'],
             appendix=appendix,
+            log_tail_lines=self.settings['log_tail_lines'],
```

Comment:
@choldgraf here is where we pass the arg to kubernetes. It’s the `--tail` arg to kubectl logs.